frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/TableDetail/Columns/Columns.tsx

```diff
@@ -16,17 +16,18 @@ type Props = {
 
 export const Columns: FC<Props> = ({ columns }) => {
   const [isClosed, setIsClosed] = useState(false)
-  const handleClose = () => {
+  const handleClose = (event: MouseEvent) => {
+    event.stopPropagation()
```

Comment:
Since the ``handleClose`` function is also used in the header, I added the above code to prevent double event firing.